staging/src/k8s.io/apiextensions-apiserver/test/integration/subresources_test.go

```diff
@@ -536,8 +536,7 @@ func TestValidateOnlyStatus(t *testing.T) {
 			if err != nil {
 				t.Fatalf("unexpected error setting .status.num: %v", err)
 			}
-			createdNoxuInstance, err = noxuResourceClient.UpdateStatus(createdNoxuInstance, metav1.UpdateOptions{})
-			if err == nil {
+			if _, err = noxuResourceClient.UpdateStatus(createdNoxuInstance, metav1.UpdateOptions{}); err != nil {
```

Comment:
I made a stupid mistake here.
Stop using `if err = xxx; err != nil` ...since here we expect nil not error.

Reply:
Fixed.